opentelemetry-metrics-exporters/src/main/java/com/cumulocity/exporters/common/OpenTelemetryExporterStrategy.java

```diff
@@ -0,0 +1,9 @@
+package com.cumulocity.exporters.common;
+
+import io.opentelemetry.api.OpenTelemetry;
+
+public interface OpenTelemetryExporterStrategy {
```

Comment:
This is the initial implementation of a Strategy pattern enabling users to select from multiple exporters